coordinator/src/main/java/org/apache/uniffle/coordinator/ServerNode.java

```diff
@@ -51,7 +52,7 @@ public class ServerNode implements Comparable<ServerNode> {
   private long startTime = -1;
   private String version;
   private String gitCommitId;
-  Map<String, RssProtos.ApplicationInfo> appIdToInfos;
+  @JsonIgnore Map<String, RssProtos.ApplicationInfo> appIdToInfos;
```

Comment:
Why do we need this?

Reply:
`RssProtos.ApplicationInfo` can't be serialized. if `appIdToInfos` is not empty, we can't see any node in the dashboard ui.

Reply:
I got it